### PR TITLE
added ashparsh-pandey

### LINKED ISSUE #5596: [Community] Member Profile: Ashparsh Pandey
**Current Behavior**

@Ashparshp has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Ashparshp as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @Ashparshp 
- Twitter: 
- LinkedIn: - Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by srujan0404**
**Description**

This PR fixes #5596

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Deepak0320Singhal** [comment]
@srujan0404 Let's discuss this on Websites's call today 6:30 PM IST. Let's add this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit)

File: src/collections/members/ashparsh-pandey/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Ashparsh Pandey
+position: Contributor
+image_path: ./ashparsh-pandey.webp
+github: Ashparshp
+twitter: ashparsh_
+linkedin: ashparsh
+location: Uttar Pradesh, India 
+bio: Hello! :wave: CS student here, passionate about coding and creating. Exploring the tech world. Let's connect and code something awesome.
```
Comment (iArchitSharma) [inline src/collections/members/ashparsh-pandey/index.mdx:9]: Remove `:wave: ` causing build issues

File: src/collections/members/ashparsh-pandey/index.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+badges: 
+    - community
```
Comment (iArchitSharma) [inline src/collections/members/ashparsh-pandey/index.mdx:11]: You are using the wrong badge
checkout his member profile - https://meshery.layer5.io/user/756c3044-ee4f-4ad6-be15-27a7967724e7?tab=badges

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
Cloud profile link is also missing!

File: src/collections/members/ashparsh-pandey/index.mdx (hunk 2 above)
Comment (srujan0404) [reply src/collections/members/ashparsh-pandey/index.mdx:11]: ok will make the changes needed

**Comment by sudhanshutech** [comment]
build is failing

**Comment by sudhanshutech** [comment]
@srujan0404 what's the update?

**Comment by srujan0404** [comment]
the badge is not available @sudhanshutech @iArchitSharma

**Comment by l5io** [comment]
🚀 Preview for commit a1d063228627cfb6fa60b14e1a3b29ce87fa3694 at: https://667eefd22d3c73522c7b6708--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**